### ISSUE TITLE
Farming 22

### ISSUE THREAD

**Issue by leo32128fs** [RivalAUT/FS19_SprayerSectionControl#35]
Hello good morning I would like your mod on FS 22, it helps a lot to make the game a little more realistic

**Comment by Lactic68** [comment]
I second this wish :) Thank you for this mod @RivalAUT 
I hope we will soon see your mod on FS22 too, by then, I wish you a merry christmas.

**Comment by sashaman** [comment]
know your game geez... 
already part of basegame 
![grafik](https://user-images.githubusercontent.com/1388895/147477960-5dd854cd-caf0-4bef-a2ff-af5772ac87a8.png)

**Comment by RivalAUT** [comment]
@sashaman But that's not the same. It does not automatically turn the section on or off depending on the ground status. It's just a simple way to change the working width.

I will convert my mod to FS22, it will take some time though.

**Comment by leo32128fs** [comment]
> ‎conheça seu jogo, nossa... ‎‎ já parte do jogo base‎ ![grafik](https://user-images.githubusercontent.com/1388895/147477960-5dd854cd-caf0-4bef-a2ff-af5772ac87a8.png)
Yes I know about this function I know my game very well , but this crude and crappy function and they added it in the game only serves to adjust the width and the flow adjustment does not work I would rather wait for the SprayerSectionControl than use this game function

**Comment by leo32128fs** [comment]
> @sashaman But that's not the same. It does not automatically turn the section on or off depending on the ground status. It's just a simple way to change the working width.
> 
> I will convert my mod to FS22, it will take some time though.

you can be sure that when it's ready I'll be the first to download it.

**Comment by smeely** [comment]
> @sashaman But that's not the same. It does not automatically turn the section on or off depending on the ground status. It's just a simple way to change the working width.
> 
> 
> 
> I will convert my mod to FS22, it will take some time though.

Any updates on the conversion? This mod was one of my favourites in fs19 and I still miss it to this day!
